common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -261,15 +261,15 @@ public boolean onTouch(final View view, final MotionEvent event) {
     }
 
     // For ClientCertAuthChallengeHandler within AADWebViewClient,
-    // the YubiKitManager needs to stop discovering Usb devices upon fragment destroy.
+    // the smartcard manager needs to stop discovering Usb devices upon fragment destroy.
     @Override
     public void onDestroy() {
         super.onDestroy();
         final String methodTag = TAG + ":onDestroy";
         if (mAADWebViewClient != null) {
-            mAADWebViewClient.stopYubiKitManagerUsbDiscovery();
+            mAADWebViewClient.stopSmartcardUsbDiscovery();
         } else {
-            Logger.error(methodTag, "YubiKitManager usb discovery not stopped due to mAADWebViewClient being null", null);
+            Logger.error(methodTag, "Usb discovery not stopped due to mAADWebViewClient being null", null);
```

Comment:
Error message exposes the name of a private variable. Would suggest something more meaningful for somebody reviewing the log who may not have source access

Reply:
This was changed to, "Fragment destroyed, but smartcard usb discovery was unable to be stopped."